.travis.yml

```diff
@@ -99,10 +101,11 @@ matrix:
         - os: linux
           env: PYTHON_VERSION=3.6
                NUMPY_VERSION=1.15
+               OPENMP_EXPECTED=False
                INSTALL_CMD='python setup.py build_ext --inplace'
                CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
-               PIP_DEPENDENCIES='sphinx pytest-doctestplus pytest-openfiles'
-               TEST_CMD='pytest --open-files --doctest-rst'
+               PIP_DEPENDENCIES='sphinx pytest-openfiles'
+               TEST_CMD='pytest --open-files'
```

Comment:
I was tempted to do this in my PR as well.  😂

Reply:
There is really no doc worth testing at this point, so... :woman_shrugging: